plugin/extraction_plugin.py

```diff
@@ -10,13 +10,14 @@ class Plugin(ExtractionPlugin):
     def plugin_info(self):
         plugin_info = PluginInfo(
             self,
-            name='your_plugin_name',
             version='0.0.0',
             description='description of your plugin',
             author=Author('Your name', 'your@email.address', 'your organisation'),
             maturity=MaturityLevel.PROOF_OF_CONCEPT,
             webpage_url='',  # e.g. url to the code repository of your plugin
-            matcher='* and $data.type=raw'
+            matcher='$data.type=raw',
+            id=PluginId('domain', 'category', 'your_plugin_name'),
```

Comment:
Can you move this line to the line before `version=`? (so it replaces the `name` argument).
I think it is makes the plugin easier identifiable (pun intended) for human users that have a quick look at the plugin.